Library/Homebrew/brew.sh

```diff
@@ -787,6 +818,7 @@ EOS
   # a popup window asking the user to install the CLT
   if [[ -n "${XCODE_SELECT_PATH}" ]]
   then
+    # TODO: this is fairly slow, figure out if there's a faster way.
     XCRUN_OUTPUT="$(/usr/bin/xcrun clang 2>&1)"
```

Comment:
@Bo98 any ideas for a faster command here? Wondering if there's anything else that can be run with `xcrun`, execute faster in the typical case but still produce the license issue otherwise 🤔

Reply:
For the first time execution, most of the slowness is typically `xcrun` itself due to searching. If you're still seeing the issue on 2nd execution, try clear the license and see if any of these reproduce:

```
$ find /usr/bin -samefile /usr/bin/clang -exec basename {} \; | sort
DeRez
GetFileInfo
ResMerger
Rez
SetFile
SplitForks
ar
as
asa
bison
c++
c++filt
c89
c99
cc
clang
clang++
clangd
cmpdylib
codesign_allocate
cpp
ctags
ctf_insert
dsymutil
dwarfdump
flex
flex++
g++
gatherheaderdoc
gcc
gcov
git
git-receive-pack
git-shell
git-upload-archive
git-upload-pack
gm4
gnumake
gperf
hdxml2manxml
headerdoc2html
indent
install_name_tool
ld
lex
libtool
lipo
lldb
llvm-g++
llvm-gcc
lorder
m4
make
mig
nm
nmedit
objdump
otool
pagestuff
pip3
python3
ranlib
resolveLinks
rpcgen
segedit
size
sourcekit-lsp
strings
strip
swift
swiftc
unifdef
unifdefall
vtool
xml2man
yacc
```

(though ideally one that's already likely to be in the cache - or in the cache soon)

There's also `defaults read /Library/Preferences/com.apple.dt.Xcode` but then you're on a parade of finding the Xcode install to extract the version from.